src/components/Menu/index.tsx

```diff
@@ -105,6 +105,20 @@ class MenuBar extends React.Component<MenuProps, _MenuState> {
     return true;
   };
 
+  checkStartEndPoints = (maze: MazeInfo): boolean => {
+    let numStart: number = 0;
+    let numEnd: number = 0;
+    // eslint-disable-next-line array-callback-return
+    Object.keys(maze).map((key: string) => {
+      // eslint-disable-next-line array-callback-return
+      maze[+key].map((value: Space) => {
+        numStart += value.type === "startpoint" ? 1 : 0;
```

Comment:
You can replace any space type string value with the enum in Spaces.types.ts

Reply:
ooo, thats a great idea, I'll refactor that before I merge